src/main/scala/io/sdkman/changelogs/LayrryMigrations.scala

```diff
@@ -0,0 +1,22 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "042")
+class LayrryMigrations {
+
+  @ChangeSet(
+    order = "001",
+    id = "001_add_layrry_candidate",
+    author = "aalmiray"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "layrry",
+      name = "Layrry",
+      description =
+        "Layrry - A Launcher and API for Modularized Java Applications.",
```

Comment:
Could you make the description a little more detailed please? This is the description used on the website as well as what a dev sees when typing `sdk list`.

Reply:
done 😄